linux/Dockerfile.linux

```diff
@@ -0,0 +1,24 @@
+FROM alpine:edge
```

Comment:
Just a meta comment; if a folder only has one `Dockerfile`, should we just leave it as `Dockerfile`, rather than `Dockerfile.<project>`?

Reply:
Sure, I did that initially for llvm-project because I expect us to dump the bootstrap version quickly. Though, I suppose we could just rewrite/modify the existing file. I guess if we ever need to resurrect the bootstrap, it's never lost, it's just buried in git history.  I suspect we may need to re-bootstrap for different host architectures though.

Reply:
If I `git mv` these to `Containerfile`, I get an error running `build.sh`:
```
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /android2/containers/linux/Dockerfile: no such file or directory
```
So what's the point of using `Containerfile` if `docker` still would need `-f Containerfile`?

Reply:
We can just move them to `Dockerfile` instead of `Containerfile` if `docker` cannot handle `Containerfile` (although I am curious how CI is working for the existing images at that point?). We can change the existing QEMU one to `Dockerfile` as well.

Reply:
> although I am curious how CI is working for the existing images at that point?

I think we [_do_](https://github.com/ClangBuiltLinux/containers/blob/5f3ebb255e7620e775ca516736c46505ef2686b3/.github/workflows/docker.yml#L46) specify a file name for the CI.

> We can change the existing QEMU one to Dockerfile as well.

Will do.

Reply:
https://github.com/ClangBuiltLinux/containers/pull/6

Reply:
done, PTAL